ast/spec.md

```diff
@@ -149,6 +151,15 @@ interface Identifier <: Expression, Pattern {
 }
 ```
 
+# PrivateName
+
+```js
+interface PrivateName <: Expression, Pattern {
```

Comment:
I'm curious to hear other's thoughts, but I'd probably have expected this to be a normal `Identifier`, rather than having a specific subtype for the private now.